src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -36,6 +38,76 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     internal static class SupportedAlgorithms
     {
+        internal static ICollection<string> EcdsaSigningAlgorithms = new Collection<string>
+        {
+            SecurityAlgorithms.EcdsaSha256,
+            SecurityAlgorithms.EcdsaSha256Signature,
+            SecurityAlgorithms.EcdsaSha384,
+            SecurityAlgorithms.EcdsaSha384Signature,
+            SecurityAlgorithms.EcdsaSha512,
+            SecurityAlgorithms.EcdsaSha512Signature
+        };
+
+        internal static ICollection<string> HashAlgorithms = new Collection<string>
+        {
+            SecurityAlgorithms.Sha256,
+            SecurityAlgorithms.Sha256Digest,
+            SecurityAlgorithms.Sha384,
+            SecurityAlgorithms.Sha384Digest,
+            SecurityAlgorithms.Sha512,
+            SecurityAlgorithms.Sha512Digest
+        };
+
+        internal static ICollection<string> RsaEncryptionAlgorithms = new Collection<string>
+        {
+            SecurityAlgorithms.RsaOAEP,
+            SecurityAlgorithms.RsaPKCS1,
+            SecurityAlgorithms.RsaOaepKeyWrap
+        };
+
+        internal static ICollection<string> RsaSigningAlgorithms = new Collection<string>
+        {
+            SecurityAlgorithms.RsaSha256,
+            SecurityAlgorithms.RsaSha384,
+            SecurityAlgorithms.RsaSha512,
+            SecurityAlgorithms.RsaSha256Signature,
+            SecurityAlgorithms.RsaSha384Signature,
+            SecurityAlgorithms.RsaSha512Signature
+        };
+
+        internal static ICollection<string> RsaPssSigningAlgorithms = new Collection<string>
+        {
+            SecurityAlgorithms.RsaSsaPssSha256,
+            SecurityAlgorithms.RsaSsaPssSha384,
+            SecurityAlgorithms.RsaSsaPssSha512,
+            SecurityAlgorithms.RsaSsaPssSha256Signature,
```

Comment:
Should this one be added to the supported algorithms wiki page?

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Supported-Algorithms

Reply:
updated wiki